### PR TITLE
[Feature] Add support for mass updating backgrounds with text

### PR DESCRIPTION
**Pull request by lostb1t**
## Description

I have a case where I want all my backgrounds to have text for the recommended rows (see https://github.com/sarendsen/replex the hero items use backgrounds)

This MR add this functionality and also adds an option to skip show episodes while batch updating. Im not sure about the yaml config though. Id rather have nested option like this

```yaml
mass_update_background:
 type: tmdb
 text: true
 skip_episodes: true
 skip_seasons: true 
```
But I implemented it as flat for now to keep in check with the rest (see operations.md). 
Let me know what ya think

## Type of Change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code was submitted to the nightly branch of the repository.

### CODE REVIEW

File: modules/operations.py [hunk 1]
```diff
@@ -511,10 +511,16 @@ def update_rating(attribute, item_attr, display):
...
                         new_background = None
+                    
+                    if self.library.mass_background_update and self.library.mass_background_update in ["tmdb_text", "tmdb_text_no_episode"] and tmdb_item:
+                        tmdb_backdrop_url = tmdb_item.backdrop_url
+                        if(len(tmdb_item.backdrops) > 0):
+                            tmdb_backdrop_url = tmdb_item.backdrops[0].url
+
                     if self.library.mass_poster_update:
-                        self.library.poster_update(item, new_poster, tmdb=tmdb_item.poster_url if tmdb_item else None)
+                        self.library.poster_update(item, new_poster, tmdb=tmdb_backdrop_url if tmdb_item else None)
```
Comment (meisnate12) [inline modules/operations.py:521]: this shouldnt change

File: modules/tmdb.py [hunk 2]
```diff
@@ -130,7 +131,7 @@ def __init__(self, tmdb, tmdb_id, ignore_cache=False):
     @retry(stop_max_attempt_number=6, wait_fixed=10000, retry_on_exception=util.retry_if_not_failed)
     def load_movie(self):
         try:
-            return self._tmdb.TMDb.movie(self.tmdb_id, partial="external_ids,keywords")
+            return self._tmdb.TMDb.movie(self.tmdb_id, partial="external_ids,keywords,backdrops")
```
Comment (meisnate12) [inline modules/tmdb.py:134]: backdrops isnt a valid endpoint to load

**Comment by meisnate12** [comment]
so the text background hasnt been added yet but i did add the ability to skip seasons/episodes in nightly

**Comment by YozoraXCII** [comment]
@sarendsen are you planning on addressing the above comments?

**Comment by lostb1t** [comment]
Sorry for the late response. this functionality is not needed anymore from my side so can be closed.

**Pull request closed without merging**